dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -503,7 +503,7 @@ class ScriptLevelsControllerTest < ActionController::TestCase
 
   test "should 404 for invalid script level for twenty hour" do
     assert_raises(ActiveRecord::RecordNotFound) do # renders a 404 in prod
-      get :show, script_id: Script::TWENTY_HOUR_ID, id: 40000
+      get :show, script_id: Script::TWENTY_HOUR_NAME, id: 40000
```

Comment:
But there are pitfalls when actions like `reports#header_stats` don't accept a name, only an ID.  In [this change](https://github.com/code-dot-org/code-dot-org/commit/7587e13acaa68dc8e04a2423faf33eccb9df0857) I tried `Script::TWENTY_HOUR_NAME` and it loads the Script with `id == 20` (!) not the Script with `name == '20-hour'`.

What's the best way to make this behavior consistent across the board?  Otherwise we end up with subtle bugs in actions that only take `script_id` as an integer.